en-us/faq.md

```diff
@@ -0,0 +1,29 @@
+# FAQ
+
+### How to install?
+
+- There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions) to install FO on the launcher that you use.
+
+### So I just copy the mods, right?
+
+- Well no because each FO instance / ZIP contains configs and a custom resource pack that is built to help the user navigate through Mod Menu. If you only copy the mods over, the configs or overrides would not apply and you would configure the mods yourself which could lead to not having the right settings to replicate OptiFine.
```

Comment:
revise the phrasing a bit
something along the lines of 
"No. Fabulously Optimized consists of mods, configs and a resource pack. If you only..."